app/serializers/api/v2/serializable_course.rb

```diff
@@ -19,6 +19,10 @@ def enrichment_attribute(name, enrichment_name = name)
         @object.start_date&.iso8601
       end
 
+      attribute :changed_at do
+        @object.changed_at&.iso8601
+      end
```

Comment:
Would it be worth exposing this as `last_published_at` so that we hide the implementation from the client?

Reply:
Updated